packages/nextjs/app/admin/_components/SubmissionCard.tsx

```diff
@@ -53,44 +60,93 @@ export const SubmissionCard = ({ submission }: { submission: Submission }) => {
     }
   };
 
+  const vote = async (newScore: number) => {
+    try {
+      if (!newScore || newScore < 1 || newScore > 10) {
+        notification.error("Wrong score");
+        return;
+      }
+
+      await postNewVote({ score: newScore });
+
+      notification.success("Voted!");
+      refresh();
+    } catch (error: any) {
+      if (error instanceof Error) {
+        notification.error(error.message);
+        return;
+      }
+      notification.error("Something went wrong");
+    }
+  };
+
+  const scoreAvg =
+    submission.votes.length > 0
+      ? (submission.votes.map(vote => vote.score).reduce((a, b) => a + b, 0) / submission.votes.length).toFixed(2)
+      : "-";
+  const currentVote = submission.votes.find(vote => vote.builder === connectedAddress);
+  const score = currentVote ? currentVote.score : 0;
+
   return (
-    <div key={submission.id} className="card bg-primary text-primary-content">
-      <div className="card-body">
-        <h2 className="card-title">{submission.title}</h2>
-        {submission.linkToRepository && (
-          <a href={submission.linkToRepository} className="link" target="_blank">
-            {submission.linkToRepository}
-          </a>
-        )}
-        <p>{submission.description}</p>
-        {submission.builder && <Address address={submission.builder} />}
-        <div className="collapse">
-          <input type="checkbox" />
-          <div className="collapse-title text-xl font-medium">{submission.comments.length} comments</div>
-          <div className="collapse-content">
-            {submission.comments?.map(comment => (
-              <div key={comment.id} className="card bg-base-200 text-base-content mb-4">
+    <div key={submission.id} className="indicator">
+      <div className="indicator-item badge badge-secondary flex flex-col p-8">
+        <div className="text-2xl font-bold">{scoreAvg}</div>
+        <div>{submission.votes.length} votes</div>
+      </div>
+      <div className="card bg-primary text-primary-content">
+        <div className="card-body">
+          <div className="flex mb-4 items-center">
+            <div className="rating">
+              <input type="radio" name={`rating_${submission.id}`} className="rating-hidden" checked={score === 0} />
+              {[...Array(10)].map((_e, i) => (
+                <input
+                  type="radio"
+                  name={`rating_${submission.id}`}
+                  className="mask mask-star"
+                  title={(i + 1).toString()}
+                  checked={score === i + 1}
+                  key={i}
+                  onClick={() => vote(i + 1)}
+                />
+              ))}
+            </div>
```

Comment:
I'm getting warnings in the console, maybe we need to use `onChange` instead of `onClick`?

```
at async Submissions (./app/admin/_components/Submissions.tsx:16:25)
Warning: You provided a checked prop to a form field without an onChange han
dler. This will render a read-only field. If the field should be mutable use de
faultChecked. Otherwise, set either onChange or readOnly.
    at input
    at div
    at div
    at div
    at div
    at div
    at SubmissionCard (webpack-internal:///(ssr)/./app/admin/_components/Submiss
ionCard.tsx:32:27)
    at Lazy
    at div
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/not-found-boundary.js:54:9)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at main
    at div
    at ScaffoldEthApp (webpack-internal:///(ssr)/./components/ScaffoldEthAppWith
Providers.tsx:42:27)
    at div
    at ModalProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainb
owkit/dist/index.js:7949:26)
    at ShowBalanceProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me
/rainbowkit/dist/index.js:3312:32)
    at TransactionStoreProvider (webpack-internal:///(ssr)/./node_modules/@rainb
ow-me/rainbowkit/dist/index.js:3780:37)
    at ModalSizeProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/r
ainbowkit/dist/index.js:3959:30)
    at I18nProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainbo
wkit/dist/index.js:2961:23)
    at WalletButtonProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-m
e/rainbowkit/dist/index.js:3942:33)
    at RainbowKitChainProvider (webpack-internal:///(ssr)/./node_modules/@rainbo
w-me/rainbowkit/dist/index.js:3280:36)
    at RainbowKitProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/
rainbowkit/dist/index.js:4658:31)
    at RainbowKitAuthenticationProvider (webpack-internal:///(ssr)/./node_module
s/@rainbow-me/rainbowkit/dist/index.js:2201:45)
    at RainbowKitSiweNextAuthProvider (webpack-internal:///(ssr)/./node_modules/
@rainbow-me/rainbowkit-siwe-next-auth/dist/index.js:14:43)
    at QueryClientProvider (webpack-internal:///(ssr)/./node_modules/@tanstack/r
eact-query/build/modern/QueryClientProvider.js:23:30)
    at SessionProvider (webpack-internal:///(ssr)/./node_modules/next-auth/react
/index.js:429:26)
    at Hydrate (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/hydrate.
js:10:13)
    at WagmiProvider (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/co
ntext.js:12:13)
    at ScaffoldEthAppWithProviders (webpack-internal:///(ssr)/./components/Scaff
oldEthAppWithProviders.tsx:88:40)
    at Lazy
    at f (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:21:24)
    at $ (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:18:62)
    at ThemeProvider (webpack-internal:///(ssr)/./components/ThemeProvider.tsx:1
3:26)
    at Lazy
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at ReactDevOverlay (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
    at HotReload (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/react-dev-overlay/hot-reloader-client.js:296:11)
    at Router (webpack-internal:///(ssr)/./node_modules/next/dist/client/compone
nts/app-router.js:155:11)
    at ErrorBoundaryHandler (webpack-internal:///(ssr)/./node_modules/next/dist/
client/components/error-boundary.js:99:9)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at AppRouter (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/app-router.js:421:13)
    at Lazy
    at Lazy
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at ServerInsertedHTMLProvider (C:\GitHub\extensions-hackathon\packages\nextj
s\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:17605)
Warning: You provided a checked prop to a form field without an onChange han
dler. This will render a read-only field. If the field should be mutable use de
faultChecked. Otherwise, set either onChange or readOnly.
    at input
    at div
    at div
    at div
    at div
    at div
    at SubmissionCard (webpack-internal:///(ssr)/./app/admin/_components/Submiss
ionCard.tsx:32:27)
    at Lazy
    at div
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/not-found-boundary.js:54:9)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at main
    at div
    at ScaffoldEthApp (webpack-internal:///(ssr)/./components/ScaffoldEthAppWith
Providers.tsx:42:27)
    at div
    at ModalProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainb
owkit/dist/index.js:7949:26)
    at ShowBalanceProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me
/rainbowkit/dist/index.js:3312:32)
    at TransactionStoreProvider (webpack-internal:///(ssr)/./node_modules/@rainb
ow-me/rainbowkit/dist/index.js:3780:37)
    at ModalSizeProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/r
ainbowkit/dist/index.js:3959:30)
    at I18nProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainbo
wkit/dist/index.js:2961:23)
    at WalletButtonProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-m
e/rainbowkit/dist/index.js:3942:33)
    at RainbowKitChainProvider (webpack-internal:///(ssr)/./node_modules/@rainbo
w-me/rainbowkit/dist/index.js:3280:36)
    at RainbowKitProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/
rainbowkit/dist/index.js:4658:31)
    at RainbowKitAuthenticationProvider (webpack-internal:///(ssr)/./node_module
s/@rainbow-me/rainbowkit/dist/index.js:2201:45)
    at RainbowKitSiweNextAuthProvider (webpack-internal:///(ssr)/./node_modules/
@rainbow-me/rainbowkit-siwe-next-auth/dist/index.js:14:43)
    at QueryClientProvider (webpack-internal:///(ssr)/./node_modules/@tanstack/r
eact-query/build/modern/QueryClientProvider.js:23:30)
    at SessionProvider (webpack-internal:///(ssr)/./node_modules/next-auth/react
/index.js:429:26)
    at Hydrate (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/hydrate.
js:10:13)
    at WagmiProvider (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/co
ntext.js:12:13)
    at ScaffoldEthAppWithProviders (webpack-internal:///(ssr)/./components/Scaff
oldEthAppWithProviders.tsx:88:40)
    at Lazy
    at f (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:21:24)
    at $ (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:18:62)
    at ThemeProvider (webpack-internal:///(ssr)/./components/ThemeProvider.tsx:1
3:26)
    at Lazy
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at ReactDevOverlay (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
    at HotReload (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/react-dev-overlay/hot-reloader-client.js:296:11)
    at Router (webpack-internal:///(ssr)/./node_modules/next/dist/client/compone
nts/app-router.js:155:11)
    at ErrorBoundaryHandler (webpack-internal:///(ssr)/./node_modules/next/dist/
client/components/error-boundary.js:99:9)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at AppRouter (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/app-router.js:421:13)
    at Lazy
    at Lazy
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at C:\GitHub\extensions-hackathon\packages\nextjs\node_modules\next\dist\com
piled\next-server\app-page.runtime.dev.js:35:381577
    at ServerInsertedHTMLProvider (C:\GitHub\extensions-hackathon\packages\nextj
s\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:17605)
Warning: You provided a checked prop to a form field without an onChange han
dler. This will render a read-only field. If the field should be mutable use de
faultChecked. Otherwise, set either onChange or readOnly.
    at input
    at div
    at div
    at div
    at div
    at div
    at SubmissionCard (webpack-internal:///(ssr)/./app/admin/_components/Submiss
ionCard.tsx:32:27)
    at Lazy
    at div
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at InnerLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:239:11)
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at NotFoundErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/not-found-boundary.js:54:9)
    at NotFoundBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/not-found-boundary.js:62:11)
    at LoadingBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/layout-router.js:313:11)
    at ErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/client/
components/error-boundary.js:128:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next
/dist/client/components/layout-router.js:151:9)
    at ScrollAndFocusHandler (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/layout-router.js:226:11)
    at RenderFromTemplateContext (webpack-internal:///(ssr)/./node_modules/next/
dist/client/components/render-from-template-context.js:15:44)
    at Lazy
    at OuterLayoutRouter (webpack-internal:///(ssr)/./node_modules/next/dist/cli
ent/components/layout-router.js:322:11)
    at Lazy
    at main
    at div
    at ScaffoldEthApp (webpack-internal:///(ssr)/./components/ScaffoldEthAppWith
Providers.tsx:42:27)
    at div
    at ModalProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainb
owkit/dist/index.js:7949:26)
    at ShowBalanceProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me
/rainbowkit/dist/index.js:3312:32)
    at TransactionStoreProvider (webpack-internal:///(ssr)/./node_modules/@rainb
ow-me/rainbowkit/dist/index.js:3780:37)
    at ModalSizeProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/r
ainbowkit/dist/index.js:3959:30)
    at I18nProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/rainbo
wkit/dist/index.js:2961:23)
    at WalletButtonProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-m
e/rainbowkit/dist/index.js:3942:33)
    at RainbowKitChainProvider (webpack-internal:///(ssr)/./node_modules/@rainbo
w-me/rainbowkit/dist/index.js:3280:36)
    at RainbowKitProvider (webpack-internal:///(ssr)/./node_modules/@rainbow-me/
rainbowkit/dist/index.js:4658:31)
    at RainbowKitAuthenticationProvider (webpack-internal:///(ssr)/./node_module
s/@rainbow-me/rainbowkit/dist/index.js:2201:45)
    at RainbowKitSiweNextAuthProvider (webpack-internal:///(ssr)/./node_modules/
@rainbow-me/rainbowkit-siwe-next-auth/dist/index.js:14:43)
    at QueryClientProvider (webpack-internal:///(ssr)/./node_modules/@tanstack/r
eact-query/build/modern/QueryClientProvider.js:23:30)
    at SessionProvider (webpack-internal:///(ssr)/./node_modules/next-auth/react
/index.js:429:26)
    at Hydrate (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/hydrate.
js:10:13)
    at WagmiProvider (webpack-internal:///(ssr)/./node_modules/wagmi/dist/esm/co
ntext.js:12:13)
    at ScaffoldEthAppWithProviders (webpack-internal:///(ssr)/./components/Scaff
oldEthAppWithProviders.tsx:88:40)
    at Lazy
    at f (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:21:24)
    at $ (webpack-internal:///(ssr)/./node_modules/next-themes/dist/index.module
.js:18:62)
    at ThemeProvider (webpack-internal:///(ssr)/./components/ThemeProvider.tsx:1
3:26)
    at Lazy
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(ssr)/./node_modules/next/dist
/client/components/redirect-boundary.js:71:9)
    at RedirectBoundary (webpack-internal:///(ssr)/./node_modules/next/dist/clie
nt/components/redirect-boundary.js:79:11)
    at ReactDevOverlay (webpack-internal:///(ssr)/./node_modules/next/dist/clien
t/components/react-dev-overlay/internal/ReactDevOverlay.js:66:9)
    at HotReload (webpack-internal:///(ssr)/./node_modules/next/dist/client/comp
onents/react-dev-overlay/hot-reloader-client.js:296:11)
    at Router (webpack-internal:///(ssr)/./node_modules/next/dist/client/compone
nts/app-router.js:155:11)
```